### ISSUE TITLE
The engine "node" is incompatible with this module. Expected version "10.15.3". Got "12.16.1"

### ISSUE THREAD

**Issue by ilovepumpkin** [IBM/carbon-addons-iot-react#1194]
The official node version being used in our product is  12.16.1. Recently when we tried to upgrade carbon-addons-iot-react to latest version, we got the the error "error carbon-addons-iot-react@2.77.3: The engine "node" is incompatible with this module. Expected version "10.15.3". Got "12.16.1".

Are there any reason that 10.15.3 has to be used? It does not work with the version higher than 10.15.3?  I would propose to change engines.node to ">=10.15.3" if possible.

**Comment by tay1orjones** [comment]
Hey there - yes, this was added in #1109.

We have not tested for node v12 #736, or v14 #1136 support yet. We've ran into unit test issues with pull requests from contributors using higher versions of node, which is why the engines flag is set to 10.15.3

That said, the notice you see is just a warning. We haven't had any reports of issues using the library being consumed in applications using higher versions of node.

I hope to have this cleared up soon.
